modules/admin/src/main/kotlin/jp/co/soramitsu/iroha2/AdminIroha2Client.kt

```diff
@@ -0,0 +1,81 @@
+package jp.co.soramitsu.iroha2
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import io.ktor.client.call.receive
+import io.ktor.client.request.get
+import io.ktor.client.statement.HttpResponse
+import io.ktor.http.ContentType
+import io.ktor.http.contentType
+import org.hawkular.agent.prometheus.text.TextPrometheusMetricDataParser
+import java.io.InputStream
+import java.net.URL
+
+open class AdminIroha2Client(
+    peerUrl: URL,
+    open var telemetryUrl: URL = URL(
+        peerUrl.protocol,
+        peerUrl.host,
+        DEFAULT_TELEMETRY_PORT,
+        peerUrl.file
+    ),
+    log: Boolean = false
+) : Iroha2Client(peerUrl, log) {
+
+    open var mapper: ObjectMapper = ObjectMapper()
+
+    /**
+     * Sends metrics request
+     */
+    suspend fun metrics(): Map<String, *> {
```

Comment:
Here `Map<String, *>` type also useless.  I think better at least to return internal representation of metrics in library `MetricFamily`. Or event better to convert to our own representation to make it easier changing metric parser library